index.html

```diff
@@ -114,13 +124,143 @@ <h2>Introduction</h2>
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-feature">Feature</dfn>  See Section 2.2 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-intermediate-synchronic-document">Intermediate Synchronic Document</dfn> See Section 11.3.1.3 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#style-value-lwsp">Linear White-Space</dfn> See Section 2.3 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-presentation-processor">Presentation processor</dfn> See Section 2.2 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-processor">Processor</dfn>  Either a Presentation processor or a Transformation processor.
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-profile">Profile</dfn>  A TTML profile specification is a document that lists all the features of TTML that are required / optional / prohibited within “document instances” (files) and “processors” (things that process the files), and any extensions or constraints.
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-related-media-object">Related Media Object</dfn> See Section 2.2 at [[TTML2]].
+      </p>
+      <p>
+        <dfn>Related Video Object</dfn> A <a>Related Media Object</a> that consists of a sequence of image frames, each a rectangular array of pixels.
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-root-container-region">Root Container Region</dfn>  See Section 2.2 at [[TTML2]].
+      </p>
+      <p>
+        <dfn>Text Alternative</dfn> As defined in [[WCAG20]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-transformation-processor">Transformation processor</dfn> See Section 2.2 at [[TTML2]].
+      </p>
+    </section>
+
+  <section id="conformance">
+    <p>
+      A <a>Document Instance</a> that conforms to the profile defined herein:
+    </p>
+    <ul style="list-style-type:disc">
+      <li>MUST satisfy all normative provisions specified by the profile;</li>
+      <li>MAY include any vocabulary, syntax or attribute value associated with a Feature whose disposition is permitted or optional in the profile;</li>
+      <li>MUST include any vocabulary, syntax or attribute value associated with a Feature whose disposition is required in the profile.</li>
+      <li>MUST NOT include any vocabulary, syntax or attribute value associated with a Feature whose disposition is prohibited in the profile.</li>
+    </ul>
+    <p class="note">
+      A <a>Document Instance</a>, by definition, satisfies the requirements of Section 3.1 at [[TTML2]], and hence a Document Instance that conforms to a profile defined herein is also a conforming TTML2 Document Instance.
+    </p>
+    <p>
+      A <a>presentation processor</a> that conforms to the profile defined in this specification:
+    </p>
+    <ul style="list-style-type:disc">
+      <li>MUST satisfy the Generic Processor Conformance requirements at Section 3.2.1 of [[TTML2]]</li>
```

Comment:
Do I understand correctly that it is normal than a "presentation processor" does not satisfy the requirements in 3.2.3?  and similarly for "transformation processor" and section 3.2.2.

Reply:
Yes, I think that's right. There is even a note about it!